ambry-clustermap/src/test/java/com/github/ambry/clustermap/MockHelixAdmin.java

```diff
@@ -398,6 +400,13 @@ long getTotalDiskCapacity() {
     return totalDiskCapacity;
   }
 
+  /**
+   * @return the number of calls to the {@link #setInstanceConfig} method.
+   */
+  public int getSetInstanceConfigCallCount() {
```

Comment:
Package private? (If this method will be called in int-test, just ignore this comment)